ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -232,7 +215,7 @@ public struct AvatarGroup: View {
                         avatarView
                             .transition(.identity)
                             .modifyIf(needsCutout, { view in
-                                view.mask(Avatar.AvatarCutout(
+                                view.mask(Avatar.CircleCutout(
```

Comment:
I think this is a good opportunity to replace the deprecated [mask(_:) modifier](https://developer.apple.com/documentation/swiftui/view/mask(_:)) with [clipShape(_:style:)](https://developer.apple.com/documentation/swiftui/path/clipshape(_:style:)).
Reference: #703